### ISSUE TITLE
Customize the output format of customized field in latex.

### ISSUE THREAD

**Issue by Aria-K-Alethia** [jgm/pandoc#4488]
Hi,

I'm using pandoc 2.1.2, trying to convert some latex files into html format.

In my latex file, there are some customized fields which have different style comparing to plain text. These fields have the following format:
    
   ```
    \begin{note}   
         content of note
    \end{note}
```

or

```
    \begin{exercise}        
         content of exercise
    \end{exercise} 
```

something like them.

I want to change the style of the text in these field, like red text color, bold , but pandoc will ignore these fields and convert all of them as 

    <p>...</p>

This problem, as discussed [here](https://groups.google.com/forum/#!topic/pandoc-discuss/HVK_vBwd71Q) , I think can only be solved by modifying the reader cause the related style information is ignored in the AST so filter and writer could do nothing with it.

One relating issue is [issue2106](https://github.com/jgm/pandoc/issues/2106), which is a reverse process.

Thank you.

**Comment by jkr** [comment]
I think this would be part of the `+styles` extension for the LaTeX reader (which is as yet unimplemented).

Basically, I take it that the logic would be some form of:

```pseudocode
if isBlock(name) and pandocDoesntRecongize(name) then
  return Div("", [], [("custom-style", name)]) contents
end
```

**Comment by mb21** [comment]
I agree that this is the same as #2106 but here for readers.

Meanwhile, you can do `pandoc -f latex+raw_tex` and customize the conversion with a filter.

**Comment by tarleb** [comment]
I believe this has been resolved via #6997.
